src/components/Search/SearchGroup.tsx

```diff
@@ -8,21 +8,14 @@ import { HighlightingResponseType, SearchResultItem } from './SearchTypes';
 const SearchGroup = ({ items, highlighting }: { items: SearchResultItem[]; highlighting: HighlightingResponseType }) => {
   return items.length > 0 ? (
     <MenuGroup>
-      {items.map(({ id, allTitle, bundle, bundle_title, abstract, relative_uri }) => (
+      {items.map(({ id, allTitle, bundle_title, abstract, relative_uri }) => (
         <MenuItem
+          className="pf-u-mb-xs"
           component={(props) => <ChromeLink {...props} href={relative_uri} />}
-          description={
-            <SearchDescription
-              highlight={highlighting[id]?.abstract}
-              bundleHighlight={highlighting[id]?.bundle_title || highlighting[id]?.bundle}
-              bundle={bundle[0]}
-              description={abstract}
-              bundleTitle={bundle_title[0]}
-            />
-          }
+          description={<SearchDescription highlight={highlighting[id]?.abstract} description={abstract} />}
           key={id}
         >
-          <SearchTitle title={allTitle} highlight={highlighting[id]?.allTitle} />
+          <SearchTitle title={allTitle} bundleTitle={bundle_title[0]} />
```

Comment:
In case we get some mangled data, we should use a safe accessor:
```suggestion
          <SearchTitle title={allTitle} bundleTitle={bundle_title?.[0]} />
```